src/main/kotlin/gomushin/backend/core/infrastructure/filter/JwtAuthenticationFilter.kt

```
@@ -0,0 +1,56 @@
+        request: HttpServletRequest,
+        response: HttpServletResponse,
+        filterChain: FilterChain
+    ) {
+        try {
+            val token = resolveToken(request)
+            if (token.isNotEmpty()) {
+                val auth = createAuthentication(token)
+                SecurityContextHolder.getContext().authentication = auth
+            }
+            filterChain.doFilter(request, response)
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)